docs/getting-started/quick-start.md

```diff
@@ -2,14 +2,195 @@
+```
+
+2. Once the database is created, we need to ensure all the pods in `my-postgresql` namespace are running (it will take few minutes):
+
+```bash
+kubectl get pod -n my-postgresql
+```
+
+You should see output something like this:
+
+```
+NAME                                          READY   STATUS    RESTARTS   AGE
+backrest-backup-hippo-9gtqf                   1/1     Running   0          13s
+hippo-597dd64d66-4ztww                        1/1     Running   0          3m33s
+hippo-backrest-shared-repo-66ddc6cf77-sjgqp   1/1     Running   0          4m27s
+```
+
+The database has been created and is empty at this stage. We now need to set its schema and we will also inject a sample data set, so we can play around with the application. 
+
+3. You can initialize the database with the schema and sample data using this
+command:
+
+```bash
+bash <(curl -s https://gist.githubusercontent.com/baijum/b99cd8e542868a00b2b5efc2e1b7dc10/raw/11e790fb1d23aa4d3ee03c260169a08b36fb25bc/init-database.sh)>
+```
+
+We have now finished to configured the database for the application. We are ready to deploy the sample application and connect it to the database. 
+
+## Deploying an application
+
+In this section, we are going to deploy the application on our kubernetes cluster. For that, we will use a deployment configuration and do the configuration of our local environment to be able to test the application.
+
+1. Deploy the `spring-petclinic-rest` app with this `Deployment` configuration:
+
+```bash
+kubectl apply -f https://gist.githubusercontent.com/baijum/b99cd8e542868a00b2b5efc2e1b7dc10/raw/11e790fb1d23aa4d3ee03c260169a08b36fb25bc/app-deployment.yaml
+```
+
+2. Let's now setup the port forwarding from the application port so we can access it from our local environment
+
+```
+kubectl port-forward --address 0.0.0.0 svc/spring-petclinic-rest 9966:80 -n my-postgresql
+```
+
+3. You should be able to open [http://localhost:9966/petclinic](http://localhost:9966/petclinic) and see a [Swagger UI][swagger] where you can play with the API.
+
+At this stage, the application is not yet connected to the database. So, if you try to play around the APIs, you'll see errors returned by the application. 
+
+For example, if you try to access the list of all pets, you can see an error like this:
+
+```
+curl -X GET "http://localhost:9966/petclinic/api/pets" -H "accept: application/json"
+
+{"className":"org.springframework.transaction.CannotCreateTransactionException","exMessage":"Could
+not open JPA EntityManager for transaction; nested exception is
+org.hibernate.exception.JDBCConnectionException: Unable to acquire JDBC
+Connection"}
+```
+
+Now, we are going to see how you can use Service Binding to easily connect the application to the database.
+
+## Connecting the application to the database
+
+**Explanations about how one would do without Service binding.**
```

Comment:
@baijum : We need to put a bit more explanation on this. It's important to callout the difference we are driving the reader into.

Reply:
Done. 1c27e60b4ef80faf802e6913a35b59dcc746f62a